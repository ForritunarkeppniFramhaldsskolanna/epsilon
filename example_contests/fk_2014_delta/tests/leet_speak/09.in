GnpkXUK4o2s8LQVO3n6luILNKJGJG45M59aKJHJFX
