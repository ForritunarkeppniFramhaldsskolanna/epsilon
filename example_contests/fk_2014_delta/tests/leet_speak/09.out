6npk><|_||<4o2s8|_(,)|/03n6lu1|_|\||<_|6_|645|\/|59a|<_||-|_||=><
